config/main.py

```diff
@@ -2969,6 +2969,56 @@ def reset(ctx, interface_name):
     cmd = "sudo sfputil reset {}".format(interface_name)
     clicommon.run_command(cmd)
 
+#
+# 'mpls' subgroup ('config interface mpls ...')
+#
+
+@interface.group()
```

Comment:
please move to mpls.py

Reply:
All subgroups of interface.group() are located within main.py.  I believe they should remain in the same file.